model.proto

```diff
@@ -63,6 +63,12 @@ message HeadOptions {
   // Set the head convolution type. Support 'depthwise_separable_conv' and
   // 'standard_conv'
   optional string head_conv_type = 3 [default = 'depthwise_separable_conv'];
+  // Set the maximum value of sigmoid. When sigmoid_max > sigmoid_min, the head
```

Comment:
Discussion for field names.

Maybe a better name for these fields is something like
{min,max}_value_after_activation
because theoretically, they can be used for any type of activation (not just sigmoid).
(any proposal for other name?)
What do you think?

Additionally, please update the comment (saying that these two fields are used to shift the activation values) and probably also mention one example (which is the depth prediction you are adding).